third-party/linshare/src/test/java/org/apache/james/linshare/LinshareFixture.java

```diff
@@ -65,23 +65,28 @@ public String getPassword() {
     Credential EXTERNAL_1 = new Credential("external1@linshare.org", "password1");
     Credential EXTERNAL_2 = new Credential("external2@linshare.org", "password2");
 
+    Credential TECHNICAL_ACCOUNT = new Credential("technical@linshare.org", "technical");
+    Credential ADMIN_ACCOUNT = new Credential("root@localhost.localdomain", "adminlinshare");
+
     List<Credential> USER_CREDENTIALS = ImmutableList.of(
-        USER_1,
-        USER_2,
-        USER_3,
-        USER_4,
-        USER_5,
-        USER_6,
-        USER_7,
-        LINAGORA,
-        EXTERNAL_1,
-        EXTERNAL_2);
+            USER_1,
+            USER_2,
+            USER_3,
+            USER_4,
+            USER_5,
+            USER_6,
+            USER_7,
+            LINAGORA,
+            TECHNICAL_ACCOUNT,
```

Comment:
You are treating the technical account as a "normal" user account here. I believe it's worng.